lib/swagger-mock-validator/mock-parser/pact/pact-parser.ts

```diff
@@ -1,8 +1,6 @@
 import _ from 'lodash';
```

Comment:
Many of these changes are caused by running `prettier` through the code, which didn't exist before

Reply:
Could do... I was trying to follow the pattern of "updating as the files changed", but maybe the number of files is small enough, that a big bang is ok...